addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties

```diff
@@ -56,14 +56,14 @@ exim.importurls.warn.scheme = "{0}" does not have a scheme.
 
 exim.menu.copyurls.popup = Copy URLs
 exim.menu.export = Export
-exim.menu.export.context.urls = Export URLs for Context(s)
-exim.menu.export.messages.popup = Export Messages to File...
+exim.menu.export.context.urls = Save URLs...
```

Comment:
It seems the changes here led to duplicate menu items under Export,

Before
![image](https://github.com/user-attachments/assets/44299434-b64f-4e97-a0a8-29dcf4be1a4d)

After
![image](https://github.com/user-attachments/assets/5991f1de-eb7f-450f-9292-12f958275598)

Reply:
Given that's already in the context menu I'd drop it.

Reply:
These are automatically removed from the Export menu and shown in the context menu when there are URLs in the Sites Tree. I guess that's another bug (the same component cannot be in two menus?)

I'm okay with dropping them, but we should be consistent. The other items under Export can also move into relevant context menus.

For now I would prefer duplicating the PopupMenuItem component with different names and adding to both the context menu as well as the Export menu (e.g. `Save URLs` and `Save Context URLs`).

Reply:
That sounds like a core bug.

I'm fine with that but don't really see the advantage of duplicating them when they are already available in the correct context menu.

Reply:
There's also a duplication issue in the Site's Tree (and maybe history?) context menu, under "Include in Context" there is now always two "New Context" entries.

Reply:
See also #6012.